### ISSUE TITLE
Compile error on previously valid Sass variable syntax

### ISSUE THREAD

**Issue by koppen** [sass/libsass#2711]
Trying to compile the following Sass-syntax snippet (the `body` selector is only there to ensure we see something output), causes an error in libsass:

    Error: Invalid CSS after "$foo:#123": expected 1 selector or at-rule, was "{}"

However DartSass - and Ruby Sass before that - compiles it as expected.

## input.sass

[todo]: # (always test and report with scss syntax)
[todo]: # (use sass only when results differ from scss)

```sass
$foo:#123
body
  color: $foo
```

## Actual results

[libsass 3.5.3][1]
```css
Invalid CSS after "$foo:#123": expected 1 selector or at-rule, was "{}" on line 1 at column 10
```

## Expected result

dart sass 1.6.2
```css
body {
  color: #123;
}
```

ruby sass 3.5.6
```css
body {
  color: #123;
}
```

version info:
```cmd
libsass         3.5.3   (Sass Compiler) [C/C++]
```

[1]: http://libsass.ocbnet.ch/srcmap/#JGZvbzojMTIzCmJvZHkKICBjb2xvcjogJGZvbwoK

**Comment by nschonni** [comment]
If you add a space between `:` and `#` it works. This would be an issue with sass2scss

**Comment by koppen** [comment]
Yes, that is correct. So sass2scss is a library libsass relies on? Should I report this upstream then?
